### PR TITLE
Ref#43692 update product

### PR DESCRIPTION
**Pull request by hoangtrungthong**
## Related Tickets:
- [#43692](https://edu-redmine.sun-asterisk.vn/issues/43692)

## WHAT this PR does?
update product

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by hoangtrungthong** [comment]
ready

**Comment by hoangtrungthong** [comment]
ready

File: app/Http/Controllers/Admin/ProductController.php [hunk 1]
```diff
@@ -92,9 +114,47 @@ public function edit(Product $product)
...
+                    $img = uploadFile('files', config('path.PRODUCT_UPLOAD_PATH'), $request, $file);
+                    $product->productImages()->update([
+                        'path' => $img,
+                    ]);
+                }
+            }
+
+            DB::commit();
+
+            return redirect()->route('admin.products.index')->withSuccess('Success');
```
Comment (ghost) [inline app/Http/Controllers/Admin/ProductController.php]: localization message success nhé :3

File: resources/lang/vi/passwords.php [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    | that are given by the password broker for a password update attempt
+    | has failed, such as for an invalid token or invalid new password.
+    |
+    */
+
+    'reset'     => 'Mật khẩu mới đã được cập nhật!',
+    'sent'      => 'Hướng dẫn cấp lại mật khẩu đã được gửi!',
+    'throttled' => 'Vui lòng đợi trước khi thử lại.',
+    'token'     => 'Mã khôi phục mật khẩu không hợp lệ.',
+    'user'      => 'Không tìm thấy người dùng với địa chỉ email này.',
```
Comment (ghost) [inline resources/lang/vi/passwords.php]:
```suggestion
    'reset' => 'Mật khẩu mới đã được cập nhật!',
    'sent' => 'Hướng dẫn cấp lại mật khẩu đã được gửi!',
    'throttled' => 'Vui lòng đợi trước khi thử lại.',
    'token' => 'Mã khôi phục mật khẩu không hợp lệ.',
    'user' => 'Không tìm thấy người dùng với địa chỉ email này.',
```
Sửa lại cho m mấy chỗ này nhé cho đúng convention nhé

File: resources/lang/vi/validation.php [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+    | Validation Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines contain the default error messages used by
+    | the validator class. Some of these rules have multiple versions such
+    | as the size rules. Feel free to tweak each of these messages here.
+    |
+    */
+
+    'accepted'             => 'Trường :attribute phải được chấp nhận.',
```
Comment (ghost) [inline resources/lang/vi/validation.php]: tương tự

**Comment by hoangtrungthong** [comment]
ready

**Pull request merged**